platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/SecureCxfClientFactory.java

```diff
@@ -270,6 +288,21 @@ public T getClientForSubject(Subject subject) {
             }
         }
 
+        try {
```

Comment:
This auditing block is large and makes the flow of the method less clear. Might be good to pull it out to a private method.